models/cam-fv/readme.rst

```diff
@@ -226,635 +332,554 @@ terminating the namelist.
-existing 2-degree ensemble, so see the `Continuing after the first cycle`_
-section to start from it, instead of a single state. If you use this, ignore any
-existing inflation restart file and tell DART to make its own in the first cycle
-in ``input.nml``:
+Assimilation using CAM and WACCM should generally use one of DART's
+adaptive inflation algorithms.  
+As of 2021 these are ``inf_flavor`` = 2 (a widely used and tested option)
+and flavor 5 (similar to 2, but enhanced by the use of an gamma distribution
+instead of a normal distribution).
+"Prior" inflation is generally a better choice than "posterior",
+so set ``input.nml:filter_nml:``
 
 .. code-block:: fortran
 
-   inf_initial_from_restart    = .false.,                 .false.,
-   inf_sd_initial_from_restart = .false.,                 .false.,
+   inf_initial_from_restart    = .true.,   .false.
+   inf_sd_initial_from_restart = .true.,   .false.
 
-In any case, make the following changes (or similar) to convert from a CAM setup
-to a WACCM setup. ``CESM1_2_1_setup_hybrid``:
+For the first cycle, if you have inflation restart files,
+you should stage those in the $RUNDIR where the other restart files
+will be staged, with names which include "dart.rh.cam_output_priorinf_mean"
+and "dart.rh.cam_output_priorinf_sd" in them, so that assimilate.csh will find them.
+If you don't have restart files, set ``*initial_from_restart`` to .false.
+and assimilate.csh will create inflation restart files
+using the values in ``inf_initial`` and ``inf_sd_initial``.
 
-.. code-block::
-
-   setenv compset     F_2000_WACCM
-   setenv resolution  f19_f19  
-   setenv refcase     FV1.9x2.5_WACCM4
-   setenv refyear     2008
-   setenv refmon      12
-   setenv refday      20
-
-and the settings within ``input.nml``:
-
-.. code-block::
-
-   vert_normalization_scale_height = 2.5
-   vert_coord                = 'log_invP'
-   highest_obs_pressure_Pa   = .001,
-   highest_state_pressure_Pa = .01,
-
-If built with the SE dy-core (warning; experimental), then 4 levels will have
-extra diffusion.
-
-If there are problems with instability in the WACCM foreasts, try changing some
-of the following parameters in either the user_nl_cam section of the setup
-script or input.nml.
-
--  The default div24del2flag in WACCM is 4. Change it in the setup script to
+Perturbed Ensemble
+~~~~~~~~~~~~~~~~~~
 
-   .. code-block::
+A multidriver configuration of CAM needs an ensemble of initial condition files
+for each active component in order to start a hindcast.
+The set of files must include, at a minimum, CAM initial files and CLM restart files.
+Usually CICE is also active, and other components may be,
+which need their own restart files.
+If there is no suitable initial ensemble for starting the ensemble hindcast,
+one can be generated from a single model state
+by linking it into suitably named files 
+(see `link_ens_to_single.csh <../CESM/shell_scripts/link_ens_to_single.csh>`_)
```

Comment:
broken link

Reply:
I don't know why it's broken.  The file exists in that location in my branch.
The syntax looks right to me.

Reply:
Doh!  It's a script, so it can't display as a web page.  I'll replace with some other kind of reference.